log-viewer/modules/components/QueryRow.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 FinancialForce.com, inc. All rights reserved.
+ */
+import { LitElement, html, css } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+@customElement('query-row')
+class QueryRow extends LitElement {
+  @property({type: Number}) instances = 0;
+  @property({type: Number}) rowCount = 0;
+  @property({type: String}) query = '';
+
+  static get styles() {
```

Comment:
interesting, does lit support styles being in a separate file?

Reply:
I think it does, I was really just trying out the "recommended" approach but have not fully got through all the styling options yet.